static/templates/draft.hbs

```diff
@@ -32,11 +32,11 @@
                 <div class="messagebox-content">
                     <div class="message_top_line">
                         <div class="draft_controls">
-                            <i class="fa fa-pencil fa-lg restore-draft" aria-hidden="true" data-toggle="tooltip" title="{{t 'Restore draft' }}"></i>
-                            <i class="fa fa-trash-o fa-lg delete-draft" aria-hidden="true" data-toggle="tooltip" title="{{t 'Delete draft' }} (Backspace)"></i>
+                            <i class="fa fa-pencil fa-lg edit-draft tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Restore draft' }}"></i>
+                            <i class="fa fa-trash-o fa-lg delete-draft tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Delete draft' }} (Backspace)"></i>
                         </div>
                     </div>
-                    <div class="message_content rendered_markdown restore-draft" data-toggle="tooltip" title="{{t 'Restore draft' }}">{{rendered_markdown content}}</div>
+                    <div class="message_content rendered_markdown restore-draft tippy-zulip-tooltip" data-tippy-content="{{t 'Restore draft' }}">{{rendered_markdown content}}</div>
```

Comment:
The tooltip appearing over "Restore draft" looks really weird.  I'm not sure if it's a good idea to have a tooltip on these in general in this form.  Maybe worth a #design thread.